tests/unit/ext/switches/brocade.py

```diff
@@ -205,15 +207,17 @@ def test_get_mode(self, switch):
             response = switch._get_mode(INTERFACE1)
             assert response == 'trunk'
 
-    def test_apply_networking(self, switch, nic, network):
+    def test_modify_port(self, switch, nic, network):
         # Create a port on the switch and connect it to the nic
         port = model.Port(label=INTERFACE1, switch=switch)
-        nic.port = port
+        port.nic = nic
```

Comment:
Why the reversal here? (setting the nic of a port rather than the port of a nic)
is it just because this is the 'modify_port' function?